packages/eslint-plugin-pf-codemods/lib/rules/nav-list-variant.js

```diff
@@ -0,0 +1,39 @@
+const { getPackageImports } = require('../helpers');
+
+// https://github.com/patternfly/patternfly-react/pull/4225
+module.exports = {
+  create: function(context) {
+    const imports = getPackageImports(context, '@patternfly/react-core')
+        .filter(specifier => ['NavList', 'Nav'].includes(specifier.imported.name));
+    const navImport = imports.find(imp => imp.imported.name === 'Nav');
+    const NavListImport = imports.find(imp => imp.imported.name === 'NavList');
+
+    return !navImport || !NavListImport ? {} : {
```

Comment:
What if NavList is used in a separate file without the Nav import? Then the rule won't run right

Reply:
That is correct.